### PR TITLE
AEROGEAR-1964: Setup testing framework for the IntelliJ plugin

### PR DESCRIPTION
**Pull request by JameelB**
## Description
We need to setup TestNG in order to write tests for the IntelliJ plugin. This needs to be configured to work with gradle. 

## Task
- [x] Setup TestNG with Gradle

## Related Notes
Related JIRA Ticket - https://issues.jboss.org/browse/AEROGEAR-1964
[TestNG](http://testng.org/doc/)
[Sample Gradle project with TestNG](https://github.com/dev9com/gradle-example)

### CODE REVIEW

**Comment by maleck13** [comment]
@JameelB I am ok with this change. I am adding some tests for the cli at the moment

**Comment by maleck13** [comment]
@JameelB going to merge this change so I can make use of it

File: build.gradle [hunk 1]
```diff
@@ -22,6 +22,18 @@ configurations {
 dependencies {
     ideaSdk fileTree(dir: 'lib/sdk/', include: ['*/lib/*.jar'])
     testCompile group: 'junit', name: 'junit', version: '4.12'
+
+    testCompile 'org.testng:testng:6.8.21'
```
Comment (JameelB) [inline build.gradle]: @maleck13 The only dependency needed should just be the testNG.

Comment (maleck13) [reply build.gradle]: is there a way to get to install? Running gradlew test fails with:
```
/Users/kelly/projects/src/github.com/aerogear/aerogear-mobile-intellij-plugin/src/test/java/org/aerogear/plugin/intellij/mobile/api/MobileAPITest.java:6: error: cannot find symbol
    @Before
     ^
  symbol:   class Before
  location: class MobileAPITest
1 error
```

**Pull request merged**